inc/thorvg.h

```diff
@@ -767,31 +767,40 @@ class TVG_API RadialGradient final : public Fill
     ~RadialGradient();
 
     /**
-     * @brief Sets the radial gradient bounds.
+     * @brief Sets the radial gradient attributes.
      *
-     * The radial gradient bounds are defined as a circle centered in a given point (@p cx, @p cy) of a given radius.
+     * The radial gradient is defined by the end circle with a center (@p cx, @p cy) and a radius @p r and
+     * the start circle with a center/focal point (@p fx, @p fy) and a radius @p fr.
+     * The gradient will be rendered such that the gradient stop at an offset of 100% aligns with the edge of the end circle
+     * and the stop at an offset of 0% aligns with the edge of the start circle.
      *
-     * @param[in] cx The horizontal coordinate of the center of the bounding circle.
-     * @param[in] cy The vertical coordinate of the center of the bounding circle.
-     * @param[in] radius The radius of the bounding circle.
+     * @param[in] cx The horizontal coordinate of the center of the end circle.
+     * @param[in] cy The vertical coordinate of the center of the end circle.
+     * @param[in] r The radius of the end circle.
+     * @param[in] fx The horizontal coordinate of the center of the start circle.
+     * @param[in] fy The vertical coordinate of the center of the start circle.
+     * @param[in] fr The radius of the start circle.
      *
-     * @retval Result::InvalidArguments in case the @p radius value is zero or less.
+     * @retval Result::InvalidArguments in case the radius @p r or @p fr value is negative.
      *
-     * @note In case the @p radius is zero, an object is filled with a single color using the last color specified in the colorStops().
+     * @note In case the radius @p r is zero, an object is filled with a single color using the last color specified in the colorStops().
+     * @note If you choose not to specify a focal point, ensure that @p fx and @p fy are the same as @p cx and @p cy, and set @p fr to 0.
      */
-    Result radial(float cx, float cy, float radius) noexcept;
```

Comment:
We can likely specify the usage of the focal:
```
By adjusting the position and size of the focal point, you can create various visual effects, such as focusing the gradient closer to one edge or generating more complex shading patterns. If you don't need a focal effect, set the focal point values to be the same as the center with a zero radius.
```

Reply:
please check now, changed in thorvg.h and capi version as well.